### ISSUE TITLE
Bootable??

### ISSUE THREAD

**Issue by GuaiYiHu** [javichampi/device_qcom_msm8960_caf#1]

**Comment by javichampi** [comment]
No. It doesn´t boot at all. Inspired by pexcn sources for CM12.1, I'm trying to use caf mm sources for msm8960 but I think it`s not a good idea. I´m rolling back to papjul, sndvapps, etc stuff.

**Comment by GuaiYiHu** [comment]
Good luck.
